pkg/tablecodec/tablecodec.go

```diff
@@ -1415,6 +1438,16 @@ func (v *TempIndexValueElem) DecodeOne(b []byte) (remain []byte, err error) {
 			v.Handle, _ = kv.NewCommonHandle(b[:hLen])
 		}
 		b = b[hLen:]
+		if b[0] == TempIndexKeyTypePartitionIDFlag {
```

Comment:
Is this only needed here, and not also in `case TempIndexValueFlagNormal:`?

Reply:
Yes, only delete will include handle. https://github.com/pingcap/tidb/pull/57017/files#diff-d38a794eb5c0ced0fe2e07bc0af6bd8624e360f4c5055b6c7075d62afac48569R1310

Reply:
Also https://github.com/pingcap/tidb/pull/57017/files#diff-d38a794eb5c0ced0fe2e07bc0af6bd8624e360f4c5055b6c7075d62afac48569L1338